stable/database/templates/_helpers.tpl

```diff
@@ -185,12 +185,53 @@ Return options as $key $value
 {{- end -}}
 
 {{/*
-Return the hotcopy group
+Return the hotcopy group prefix
 */}}
-{{- define "hotcopy.group" -}}
-{{ default .Values.cloud.cluster.name .Values.database.sm.hotCopy.backupGroup }}
+{{- define "hotcopy.groupPrefix" -}}
+{{ default .Values.cloud.cluster.name .Values.database.sm.hotCopy.backupGroupPrefix }}
 {{- end -}}
 
+{{/*
+Return the hotcopy cronjob schedule by backup group and hot copy type. It will take 
+into account any schedule overwrites configured per backup group.
+*/}}
+{{- define "hotcopy.group.schedule" -}}
+  {{- $scheduleProp := printf "%sSchedule" .hotCopyType -}}
+  {{- $schedule := index .Values.database.sm.hotCopy ( print $scheduleProp ) -}}
+  {{- $overwrite := "" -}}
+  {{- if eq .hotCopyType "journal" -}}
+    {{- $schedule = .Values.database.sm.hotCopy.journalBackup.journalSchedule -}}
+  {{- end -}}
+  {{- if .Values.database.sm.hotCopy.backupGroups -}}
+    {{- $group := (index .Values.database.sm.hotCopy.backupGroups ( print .backupGroup )) -}}
+    {{- if $group -}}
+      {{- $overwrites := $group.overwrites -}}
+      {{- if $overwrites -}}
+        {{- $overwrite = index $overwrites ( print $scheduleProp ) -}}
+      {{- end -}}
+    {{- end -}}
+  {{- end -}}
+{{ default $schedule $overwrite }}
+{{- end -}}
+
+
+{{/*
+Return labels for a specific backup group. If there is no user-defined backup
+groups, return the pod name of a single HCSM by extracting the pod ordinal from
+the automatically generated backup group name. Otherwise return the configured 
+backup group labels or empty value (representing all HCSMs in the database).
+*/}}
+{{- define "hotcopy.group.labels" -}}
+  {{- if .Values.database.sm.hotCopy.backupGroups -}}
+    {{- $group := index .Values.database.sm.hotCopy.backupGroups (print .backupGroup) -}}
+    {{- if $group -}}
+{{- default "" $group.labels -}}
+    {{- end -}}
+  {{- else -}}
+    {{- $groupPrefix := include "hotcopy.groupPrefix" . -}}
+{{ printf "pod-name sm-%s-hotcopy%s" (include "database.fullname" .) (trimPrefix $groupPrefix .backupGroup) }}
```

Comment:
Same comment here. I think you should be able to indent normally and use `{{- printf ... }}`.